### ISSUE TITLE
[bitnami/keycloak] "keycloak/templates/ingress.yaml" at <.Values.ingress.servicePort>: nil pointer evaluating interface 

### ISSUE THREAD

**Issue by Ognian** [bitnami/charts#5758]
**Which chart**:
bitnami/keycloak version 2.3.0

**Describe the bug**
```
helm.go:81: [debug] template: keycloak/templates/ingress.yaml:44:135: executing "keycloak/templates/ingress.yaml" at <.Values.ingress.servicePort>: nil pointer evaluating interface 
```
works reliable with version 1.2.0 (this was my last version, so the problem was introduced somewhere between 1.2.0 and 2.3.0)

**To Reproduce**
`helm upgrade ... --debug --dry-run` shows the above error

**Expected behavior**
no error
**Version of Helm and Kubernetes**:

- Output of `helm version`:

```                  
version.BuildInfo{Version:"v3.5.2", GitCommit:"167aac70832d3a384f65f9745335e9fb40169dc2", GitTreeState:"dirty", GoVersion:"go1.15.7"}
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.3", GitCommit:"1e11e4a2108024935ecfcb2912226cedeafd99df", GitTreeState:"clean", BuildDate:"2020-10-14T12:50:19Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.5+k3s2", GitCommit:"746cf4031370f443bf1230272bc79f2f72de2869", GitTreeState:"clean", BuildDate:"2020-12-18T01:41:55Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"linux/amd64"}
```

**Additional context**
ingress is enabled

**Comment by rafariossaa** [comment]
Hi,
  I would suggest you to do incremental version updates. 
  Could you try to upgrading 1.2.0 -> 1.4.3 -> 2.0.0 -> 2.3.0 ?

**Comment by MrKrabat** [comment]
I want to install keycloak (no upgrade) and have the same issue.

It happens if you add additional hosts:
```
helm install keycloak bitnami/keycloak --set ingress.enabled=true --set ingress.extraHosts[0].name=test.local --set ingress.extraHosts[0].path=/ --dry-run
Error: template: keycloak/templates/ingress.yaml:44:135: executing "keycloak/templates/ingress.yaml" at <.Values.ingress.servicePort>: nil pointer evaluating interface {}.ingress
```

**Comment by MrKrabat** [comment]
And it got fixed by #5936 thank you :)

**Comment by marcosbc** [comment]
Thanks for the update! That's great to hear. I proceed to close this issue.
